gadget/ondisk.go

```diff
@@ -124,8 +100,11 @@ type OnDiskStructure struct {
 
 	// Node identifies the device node of the block device.
 	Node string
-	// CreatedDuringInstall is true when the structure has properties indicating
-	// it was created based on the gadget description during installation.
+	// CreatedDuringInstall is true when the structure has a fs label that
+	// matches on the partitions that is created during install mode. It is
+	// expected that this flag is not used for any destructive operations until
+	// it is independently verified that the on disk structure actually matches
+	// the expected gadget structure via ensureLayoutCompatibility for example.
 	CreatedDuringInstall bool
```

Comment:
with the new definition this flag is just dangerous, I remember we thought it was hard to avoid, I would still prefer if we tried not to need it, and the relevant code did tests or carried info locally

Reply:
this flag is now removed as it was as you say dangerous to use